contracts/Strategy.sol

```diff
@@ -399,10 +402,6 @@ contract Strategy is BaseStrategy {
         );
     }
 
-    // this allows us to also sell bpt externally
-    function sellBpt(uint256 _amountBpts) external onlyVaultManagers {
-        _sellBpt(_amountBpts);
-    }
```

Comment:
should put this back along with any other external functions deleted and implement the modifier trick. It'll save about ~4% bytecode size. The external functions are pretty important and have saved us in the past

Reply:
https://github.com/tonkers-kuma/strategy-ssb/blob/06695d0403a69286cc8f4780b3ddb12274465e02/contracts/Strategy.sol#L41-L48

Reply:
Adding this and bringing back the external functions